### ISSUE TITLE
Support RPM to RCF (g force)

### ISSUE THREAD

**Issue by camilogarciabotero** [sharkdp/insect#271]
Hi all,

Thanks for working on this project. I was wondering if there is any way to support the conversion from RPM to RCF (relative centrifuge force) or a.k.a G-force. 

It is common to find lab (biology) guide articles and methods that use RCF as the standard unit to describe centrifuge values. Yet, many lab centrifuges display by default the RPM value.

 Here is a simple way to represent the unit conversion:

RCF = 1.12 x Radius x (rpm/1000)2

The _Radius_ is conventionally the centrifuge rotor radius, which tends to vary according to the centrifuge's brand and is represented in mm in the aforementioned formula. [Here](http://insilico.ehu.es/mini_tools/rcf_rpm.php) a simple web calculator.

**Comment by sharkdp** [comment]
So the "proper" way to [compute this with SI units](https://en.wikipedia.org/wiki/Centrifuge#Mathematical_description) would be via

![image](https://user-images.githubusercontent.com/4209276/149617471-a154860f-b2ad-4204-a2e1-b7c4c149ff24.png)

where ![image](https://user-images.githubusercontent.com/4209276/149617483-52b26640-8bb6-4b72-9de7-ed8d09035d02.png) is the angular velocity, which we can compute in insect via

```
> omega = 2pi * 1000rpm
```

we can then set the radius:

```
> r = 100mm
```

and finally compute rcf in units of 'g' (in insect: `gravity`):

```
> r * omega^2 / gravity

   = 111.824
```

you could define your own function that takes two arguments:
```
> rcf(radius, rot_speed) = radius * (2pi rot_speed)^2 / gravity
```
which could then be used as:
```
> rcf(100mm, 1000rpm)
   = 111.824
```

**Comment by camilogarciabotero** [comment]
Hey @sharkdp 

Thanks so much for showing this way to calculate the _RFC_ via insect. I'll definitely save this to my terminal `~/.insectrc`

**Comment by camilogarciabotero** [comment]
Hi @sharkdp 

It's me again. Is the `.insectrc` feature available? It seems that the `.insectrc` is not being parsed in the insect env once it startss up.

**Comment by sharkdp** [comment]
Sorry, I forgot. The feature has not yet been released. You can try to use the latest version from master. I tried to release a new version but have problems with npm (again). I currently don't have the time to look into this further. Any help would be appreciated.

**Comment by sharkdp** [comment]
> It's me again. Is the `.insectrc` feature available? It seems that the `.insectrc` is not being parsed in the insect env once it starts up.

it's available now in v5.7.0
